src/NUnitFramework/tests/Assertions/ConditionAssertTests.cs

```diff
@@ -220,7 +220,7 @@ public void IsNotEmptyFailsOnEmptyIEnumerable()
             var expectedMessage =
                 "  Expected: not <empty>" + Environment.NewLine +
                 "  But was:  <empty>" + Environment.NewLine;
-            var ex = Assert.Throws<AssertionException>(() => Assert.IsNotEmpty((IEnumerable)new int[0]));
+            var ex = Assert.Throws<AssertionException>(() => Assert.IsNotEmpty((IEnumerable)Array.Empty<int>()));
```

Comment:
Even though your change is correct, a better version here would be:

```suggestion
            var ex = Assert.Throws<AssertionException>(() => Assert.IsNotEmpty(Enumerable.Empty<int>()));
```

You will also have to add a `using System.Linq;` to the file.

Reply:
Done